### DISCUSSION TITLE
Student Account Access

### DISCUSSION THREAD

**Discussion by Anks147** [community/community#127496] [GitHub Education]
### Hi everyone,

Does anyone know exactly how much time it takes to get approval on your Student account access?
I have tried several times to upload my id card but it keeps on rejecting it and finally I was when able to get past it I got a pop up saying my case is under consideration for the approval. Does anyone else here face the same situation ?

**Comment by ktauchathuranga** [comment]
it's usually takes about 1-2 days.
but sometimes id isn't enough. 
when you try to resubmit it will sometimes say you have give them another related document and you have to a photo (cant upload a photo)
just keeping in eye on any warnings when you try to reupload the documents. sometimes its very confusing. 
Happy coding... ✨️

**Comment by moxer-mmh** [comment]
for me it was about 1 day i think
